index.html

```diff
@@ -39,14 +46,548 @@ <h1>I'm Emeka Glad to see you!</h1>
+
+                <li>
+
+                    <a class="textdecoration font3" href="#">2015</a>
+
+                </li>
+
+            </ul>
+
+            <p>
+                A daily selection of privately personalized reads; no accounts or sign-ups required.
+            </p>
+
+            <ul class="gridbox1">
+
+                <li class="background1">
+
+                    <a class="textdecoration font4" href="#">html</a>
+
+                </li>
+
+                <li class="background1">
+
+                    <a class="textdecoration font4" href="#">css</a>
+
+                </li>
+
+                <li class="background1">
+
+                    <a class="textdecoration font4" href="#">javaScript</a>
+
+                </li>
+
+            </ul>
+
+            <button type="submit">
+
+                <a class="font5 textdecoration" href="#">See Project</a>
+
+            </button>
+
+        </div>
+
+    </section>
+
+
+    <section class="section6">
+
+        <div>
+
+            <h3>
+
+                <a class="textdecoration font6" href="#">About Myself</a>
+
+            </h3>
+
+            <p>
+                Hello I'm a software developer! I can help you build a product, feature or website Look through some of
+                my work and experience! if you like what you see and have a project you need coded, don't hesistate to
+                contact me.
+            </p>
+
+            <h5>LET'S CONNECT</h5>
+
+            <ul>
+
+                <li>
+
+                    <a href="#"><img src="./img/angular.jpg" alt="angular_image"></a>
+
+                </li>
+
+                <li>
+
+                    <a href="#"><img src="./img/github.jpg" alt="github_image"></a>
+
+                </li>
+
+                <li>
+
+                    <a href="#"><img src="./img/Linkedin.jpg" alt="Linkedin_image"></a>
+
+                </li>
+
+                <li>
+
+                    <a href="#"><img src="./img/mee.jpg" alt="mee_image"></a>
+
+                </li>
+
+                <li>
+
+                    <a href="#"><img src="./img//twitter.jpg" alt=""></a>
+
+                </li>
+
             </ul>
 
+            <button type="submit">
+
+                <a class="font5 textdecoration" href="#">Get my resume</a>
+
+            </button>
+
+            <article class="language_article">
+
+                <header>
+
+                    <h3 class="font7">Languages</h3>
+
+                    <img src="./img/arrow.svg" alt="">
+
+                </header>
+
+                <div class="language_type">
+
+                    <a href="#">
+
+                        <img src="./img/javascript_icon.jpg" alt="">
+
+                    </a>
+
+                    <a class="font5 textdecoration" href="#">
+
+                        <label for="javaScript_image">JavaScript</label>
+
+                    </a>
+
+                </div>
+
+                <div class="language_type">
+
+                    <a href="#">
```

Comment:
- [ ] Good job on working on the design. It looks great. However, from the project guidelines, this section should be implemented as `Nested list`. It may be difficult to explain but with examples, you can understand it. Here is [a good article ](https://www.webucator.com/article/how-to-properly-nest-lists-in-html/)that explains it 😉  
EXAMPLE 👇 
```
<ul>
  <li>Languages
    <ul>
      <li>Javascript</li>
      <li>Ruby</li>
    </ul>
  </li>
  <li>Framework
    <ul>
      <li>Bootstrap</li>
      <li>Ruby on Rails</li>
    </ul>
  </li>
 ....
</ul>
```

Please refer to this and improve your codes  and please don't use `article` tag here😃